src/pages/Register.js

```diff
@@ -44,7 +44,6 @@ export default class Register extends Component {
       await refreshTerraProfile()
     } catch (error) {
       reportError('Error registering', error)
-    } finally {
```

Comment:
What's the intent of this change? It looks like this will leave a spinner active if the call fails.

Reply:
The intent is not to try to dismiss the spinner after the component unmounts.